videos/.gitignore

```diff
@@ -0,0 +1 @@
+*.mp4
```

Comment:
i think i mean to make a gitkeep but i called it gitignore and just rolled with it